src/librustc_trans/back/symbol_export.rs

```diff
@@ -79,9 +79,7 @@ pub fn provide_local(providers: &mut Providers) {
     providers.is_exported_symbol = |tcx, id| {
         // FIXME(#42293) needs red/green to not break a bunch of incremental
         // tests
-        tcx.dep_graph.with_ignore(|| {
```

Comment:
:D

(you can probably delete the FIXME above this as well)

Reply:
`:)` Yes.